apps/testingaccessibility/studio/schemas/documents/package.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+
+export default {
```

Comment:
I want to make one small change after sleeping on this... package -> product as this is directly linked to the Stripe concept of a product.

course > product > module > section > lesson > resource

"course" at the top doesn't really matter, it's just how instructors/twitter tends to refer to it. I'd rather it be "product" but I think it makes more sense to align with stripe at that "transition"

This will likely have a `productId` field that connects it to the Prisma schema for a product.

Reply:
done ✅  it's using shorter `id` instead of `productId`. I figured it won't change much, but happy to use the longer form or maybe add a description... and yes I'm using it as a connection point to Prisma schema, works like a charm.

Reply:
I'd use productId similar to how it would be in the Prisma schema for a relationship versus "id" which would semantically represent the "id" of the Sanity instance for me.

Reply:
done 👍